lms/djangoapps/commerce/api/v1/tests/test_serializers.py

```diff
@@ -0,0 +1,36 @@
+""" Commerce API v1 serializer tests. """
+from django.test import TestCase
+
+from commerce.api.v1.serializers import serializers, validate_course_id
+
+
+class CourseValidatorTests(TestCase):
+    """ Tests for Course Validator method. """
+
+    def test_validate_course_id_with_non_existent_course(self):
+        """ Verify a validator checking non-existent courses."""
+        course_key = 'non/existing/keyone'
+
+        with self.assertRaises(serializers.ValidationError) as error:
+            validate_course_id(course_key)
+
+        self.assertEqual(
+            error.exception.messages[0],
```

Comment:
No error.message is not working. Following is error object info.
{'expected': <class 'django.core.exceptions.ValidationError'>, 'failureException': <type 'exceptions.AssertionError'>, 'exception': ValidationError([u'invalidkey is not a valid course key.']), 'expected_regexp': None}